qml/pages/OverviewPage.qml

```diff
@@ -88,7 +88,8 @@ Page {
     }
     Timer {
         id: updateSecondaryContentTimer
-        interval: 600
+        interval: 1500
+        repeat: false
```

Comment:
There's no need for this, `repeat` defaults to `false`

Reply:
And is this necessary at all, to increase the interval? This is a one-time action, it won't have any impact on power usage or anything like that.

Reply:
Well, yes, it's logically unrelated, I've just noticed it improves app startup performance.. Ok, I will redo the PR again.